src/models/eventModel.tsx

```diff
@@ -15,6 +15,16 @@ const ctx = {
   sequenceProbability: VocabularyUtils.PREFIX + "sequence-probability",
   childrenSequence: VocabularyUtils.PREFIX + "has-child-sequence",
   rectangle: DIAGRAM_PREFIX + "has-rectangle",
+  supertypes: VocabularyUtils.PREFIX + "is-derived-from",
+  hasFailureRate: {
```

Comment:
`hasFailureRate: VocabularyUtils.PREFIX + "has-failure-rate"` -- as everyhere above does not work?

Reply:
You was right. It works correct this way. Fixed.